third_problem.rb

```diff
@@ -0,0 +1,68 @@
+class ThirdProblem
+  def initialize(numbers_array)
+    @numbers = numbers_array
```

Comment:
Mixed convention. You use `camelCase` as well as `snake_case`. You should be sticking to one.

Reply:
fixed

Reply:
https://github.com/bbatsov/ruby-style-guide